paddle/fluid/platform/gpu_info.cc

```diff
@@ -33,6 +33,15 @@ int GetCUDADeviceCount() {
   return count;
 }
 
+int GetCUDAComputeCapability(int id) {
```

Comment:
Since this is the `cudaDeviceProp` of a GPU card, could we get this in init.cc file? Thus, we can cache this version number other than call it in every float16 method.

Reply:
Good point! Done.